### PR TITLE
epidemiological model addition

### PR DESCRIPTION
**Pull request by Liyubov**
I added description of SEIR model and relevant references. Maybe this could be more useful for people than just SIR model.

### CODE REVIEW

**Comment by rufuspollock** [comment]
@Liyubov this how conflicts somehow? Could you pull master, resolve the conflicts and update.

File: coronavirus.md [hunk 1]
```diff
@@ -69,12 +69,31 @@ Coming soon ...
...
+The simulations of SIR model are shown in the python script [here](https://github.com/Liyubov/heterogeneous-dynamics-on-networks/blob/master/code_network_heterogen_models/spreading_SIR.py).
+
+The main take-away message from the SIR model simulations is that we observe so-called peak of epidemic, when the number of infected people reaches its maximum. This moment of when size of epidemic outbreak is maximal depends on SIR model parameters: probability of epidemic transmission and recovery from the disease. 
+
+Another model, which has been proven to be much more realistic for some epidemics spreading models is **SEIR model** [1]. 
+The main idea of this model is that for many important infections there is a significant incubation period during which individuals have been infected but are not yet infectious themselves. During this period the individual is in a special state E (exposed), additional state to three states of parts of population in SIR model.
```
Comment (rufuspollock) [inline coronavirus.md:84]: You need double space between paragraphs - see http://playbook.datopian.com/style-guide/markdown

File: coronavirus.md [hunk 2]
```diff
@@ -69,12 +69,31 @@ Coming soon ...
...
+The main idea of this model is that for many important infections there is a significant incubation period during which individuals have been infected but are not yet infectious themselves. During this period the individual is in a special state E (exposed), additional state to three states of parts of population in SIR model. 
+
+The SEIR model studies number of people between four states: susceptible (S), exposed (E), infected (I), and resistant (R) , each being the number of people in those groups. There are several model parameters, which partially control how fast people move from being susceptible to exposed, from exposed to infected, and from infected to resistant. This model can have additional parameters; one is the background mortality, which is unaffected by disease-state, while the other is vaccination. The vaccination moves people from the susceptible to resistant directly, without becoming exposed or infected.
+The SEIR model is the most widely used tool for predicting epidemics and acting to suppress them. In 2020, the model was further refined by Richard Neuer [3].
```
Comment (rufuspollock) [inline coronavirus.md:87]: Ditto re line breaks 😉

File: coronavirus.md (hunk 1 above)
Comment (Liyubov) [reply coronavirus.md:84]: ok, tnx!

**Pull request merged**